contrib/deploy-onefuzz-via-azure-devops/deploy-onefuzz.yml

```diff
@@ -20,7 +20,9 @@
 # |ONEFUZZ_SERVICE_URL   | OneFuzz service URL. Generally the url defined in App     |
 # |                      | Registration                                              |
 # |REGION                | OneFuzz Region (prefer westus2)                           |
-# |RESOURCE_GROUP_NAME   | Resource gorup name for OneFuzz deployment                |                        
+# |RESOURCE_GROUP_NAME   | Resource gorup name for OneFuzz deployment                |
+# |VERSION               | Specify OneFuzz version or do not set to get latest       |
```

Comment:
Can you please add a third column where it can be specified if a variable is required or optional.

After that a task which determines if required variables exists otherwise fail and exit.